ksqldb-engine/src/main/java/io/confluent/ksql/structured/SchemaKTable.java

```diff
@@ -175,12 +173,11 @@ public SchemaKTable(
     if (keyFormat.isWindowed()) {
       final String errorMsg = "Implicit repartitioning of windowed sources is not supported. "
           + "See https://github.com/confluentinc/ksql/issues/4385.";
-      final String additionalMsg = forceRepartition
```

Comment:
Out of curiosity, what's the thinking for removing this conditional? Even though it's currently the case that this block can only be entered if `forceRepartition` is true, that may not be the case in the future.

If you're going to remove this, you should be able to remove the `UC_USELESS_CONDITION` suppression at the top of the method as well.

Reply:
I just blindly followed IntelliJ's recommendations. I'll change it back